src/client/kotlin/tech/thatgravyboat/skycubed/features/tablist/CompactTablist.kt

```diff
@@ -172,7 +174,29 @@ object CompactTablist {
             playerName = name
             face = player?.skin
             skyblockLevel = level.toInt()
-            contributor?.symbol?.let { contribEmblem = it }
+            // party > coop > friends > guild
+            // Todo add apis for this in sbapi
+            val isCoop = false
+            val isGuild = false
+            fun addExtraTag(mutableComponent: MutableComponent) {
```

Comment:
can probably just be replaced with string, color since all usages of it just call Text.of(string) { this.color = color }